### PR TITLE
Fix Menu Title Capitalization

### PR DESCRIPTION
**Pull request by AndySchneiderDev**
in reference to issue #739

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,17 +111,17 @@
       },
       {
         "command": "PowerShell.OpenInISE",
-        "title": "Open current file in PowerShell ISE",
+        "title": "Open Current file in PowerShell ISE",
```
Comment (rkeithhill) [inline package.json]: Shouldn't `file` be capitalized as well? Otherwise, LGTM.

Comment (AndySchneiderDev) [inline package.json]: Shoot, missed that somehow. How should i fix it?

Comment (rkeithhill) [inline package.json]: Did you edit this online or via a local fork? If a fork then `Add more commits by pushing to the develop branch on SchneiderAndy/vscode-powershell.`

**Comment by AndySchneiderDev** [comment]
Added in fix for File

**Comment by rkeithhill** [comment]
@AndySchneiderDev Thanks!

**Comment by daviwil** [comment]
Thanks a lot Andy!

**Pull request merged**